js/src/pages/Products.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { useQuery } from '@tanstack/react-query';
-import { Container, Typography } from '@mui/material';
+import { Container, Typography } from '../components/index';
```

Comment:
Path consistency is important
Use absolute in most cases

@/component or @component (this needs to be added to path in webpack first)